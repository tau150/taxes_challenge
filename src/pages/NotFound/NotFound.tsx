import { VStack, Box, Heading, Button, Text } from "@chakra-ui/react";
import { Link } from "react-router-dom";

import { ROUTES } from "@/routes/routes.types";

const NotFound = () => {
  return (
    <VStack h="100vh" justify="center">
      <Heading color="green.300">404!</Heading>
      <Heading>Something went wrong</Heading>
      <Text>The page you are looking for does not exist</Text>
      <Box mt="4">
        <Link to={ROUTES.ROOT}>
          <Button colorScheme="teal">Go to home</Button>
        </Link>
      </Box>
    </VStack>
  );
};

export default NotFound;
