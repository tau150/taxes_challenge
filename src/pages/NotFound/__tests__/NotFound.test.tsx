import { expect } from "vitest";

import NotFound from "../NotFound";

import { render } from "@/tests/render";

describe("NotFound page", () => {
  it("should match with snapshot", () => {
    const { container } = render(<NotFound />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
