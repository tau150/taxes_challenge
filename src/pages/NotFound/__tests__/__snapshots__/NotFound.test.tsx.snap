// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`NotFound page > should match with snapshot 1`] = ` `;
